UI/window-basic-main-transitions.cpp

```diff
@@ -1865,3 +1867,23 @@ int OBSBasic::GetOverrideTransitionDuration(OBSSource source)
 
 	return (int)obs_data_get_int(data, "transition_duration");
 }
+
+void OBSBasic::UpdateStudioModeSceneLabels()
+{
+	if (!IsPreviewProgramMode())
+		return;
+
+	QString preview = QString("%1 - %2").arg(
+		QTStr("StudioMode.Preview"),
+		QT_UTF8(obs_source_get_name(GetCurrentSceneSource())));
```

Comment:
I believe this could cause issues in right-to-left languages.
It'd probably be better to have something along the lines of 
```
QString preview = QTStr("StudioMode.Preview.Scene").arg(obs_source_get_name(GetCurrentSceneSource()));
```
where the string in the ini file would be
```
StudioMode.Preview.Scene="Preview - %1"
```
A separate string is required because the existing one is used in other places as well.